bundle.yml

```diff
@@ -92,9 +92,26 @@ options :
             ENABLE_DOUBLE_PRECISION=OFF
             FIELD_API_DEFINITIONS=SINGLE
 
+    - with-field-api :
+        help : Enable Field API variants to be built
+        cmake : >
+            BUILD_field_api=ON
+            ENABLE_CLOUDSC_FORTRAN_FIELD=ON
+
+    - with-mapped-fields :
+        help : Enable Field API to use mapped memory by default
+        cmake : >
+            ENABLE_FIELD_API_MAPPED_MEMORY=ON
+    
+    - with-pinned-fields :
```

Comment:
This bundle option should then be removed once pinning is made run-time togglable.